src/main/java/dev/snowdrop/jira/atlassian/ReleaseService.java

```diff
@@ -79,13 +79,12 @@ public static void createComponentIssues() {
 		Release release = (Release) pojo;
 
 		for (Component component : release.getComponents()) {
-			if (!component.getSkipCreation()) {
-				IssueInputBuilder iib = new IssueInputBuilder();
-				iib.setProjectKey(component.getJiraProject());
-				iib.setSummary(component.getJiraTitle());
-				iib.setDescription(generateIssueDescription(release, component));
-				iib.setIssueType(TASK_TYPE());
-				iib.setDueDate(toDateTime(release.getDueDate()));
```

Comment:
For minor releases, we will not contact the JIRA stakeholders as we will reuse the existing component version or the version as defined within the CVEs. So, we still need this option that you deleted

Reply:
No. Stakeholders should always be contacted. Every release we put out should be vetted by stakeholders, minor release or not.

Reply:
@zekaguilar Do you also agree that we contact the stakeholders (= create JIRA tickets within their project) for minor releases even if we know that we dont expect from them a new release of their component but instead a QE/Test validation that their component or starter will work with by example Spring Boot 2.x.y ?

Reply:
We never expect a new release. We don't care about new releases, we just want a *supported* release for our Spring Boot version for the duration of the release life time. That's why we always need to get this information from stakeholders.